### PR TITLE
feat:[CI-11620]:configure disk size using create vm api

### PR DESCRIPTION
**Pull request by HenAmar**
# Commit Checklist

Thank you for creating a pull request! To help us review / merge this can you make sure that your PR adheres as much as possible to the following.

## The Basics

If you are adding new functionality, please provide evidence of the new functionality.

### CODE REVIEW

File: command/config/config.go [hunk 1]
```diff
@@ -97,6 +97,7 @@ type (
 		Tag           string `json:"tag,omitempty" yaml:"tag"`
 		AuthToken     string `json:"auth_token,omitempty" yaml:"auth_token"`
 		GroupID       string `json:"group_id,omitempty" yaml:"group_id"`
+		Disk          string `json:"disk,omitempty" yaml:"disk"`
```
Comment (devkimittal) [inline command/config/config.go:100]: we will pass it from pool yaml?

Comment (HenAmar) [reply command/config/config.go:100]: yes, and if we don't it will go with the default.

Comment (devkimittal) [reply command/config/config.go:100]: what is the size that we are planning to put in prod?

Comment (devkimittal) [reply command/config/config.go:100]: SInce we just create one vm in a machine, we can use max memory available. also what is the default size right now?

Comment (HenAmar) [reply command/config/config.go:100]: default for sonoma is 128 and ventura is 256. I plan to put both with 256.

**Review by devkimittal — APPROVED** [review]

**Pull request merged**